public/app/app.ts

```diff
@@ -130,6 +132,7 @@ export class GrafanaApp {
       setPanelDataErrorView(PanelDataErrorView);
       setLocationSrv(locationService);
       setTimeZoneResolver(() => config.bootData.user.timezone);
+      __setDashboardSrv(getDashboardSrv());
```

Comment:
Nit: I think we should stick the the naming/coding conventions that we already have in the code base.

Reply:
I was hesitant about this one indeed. I was trying to follow the general js ideas that methods prefixed with dashes are supposed to be privates but I agree in reality it is little it does and breaks the convention. I'll change it.